llvm/docs/GitHub.rst

```diff
@@ -207,6 +207,76 @@ See more in-depth information about how to contribute in the following documenta
 * :doc:`Contributing`
 * :doc:`MyFirstTypoFix`
 
+Example Pull Request with git
+====================================
+
+Instead of using the GitHub CLI to create a PR, you can push your code to a
+remote branch on your fork and create the PR to upstream using the GitHub web
+interface.
+
+Here is an example of making a PR using git and the GitHub web interface:
+
+First follow the instructions to [fork the repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=webui#forking-a-repository).
+
+Next follow the instructions to [clone your forked repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo?tool=webui#cloning-your-forked-repository).
+
+Once you've cloned your forked repository,
+
+::
+
+  # Switch to the forked repo and create a new branch
+  cd llvm-project
+  git switch -c my_change
+
+  # Create your changes
+  $EDITOR file.cpp
+
+  # Don't forget clang-format
+  git clang-format
```

Comment:
Does it work without `-f`?

Reply:
I reused what we had above: https://github.com/llvm/llvm-project/pull/65393/files#diff-2ffdcd91784145f84fd7eba67dacb7ef2b20ea5469917531bf119e2207f77a3dL166

If we need a `-f` here then we need it above too.